dash/backend/src/modules/shared/services/email.service.ts

```diff
@@ -23,13 +24,17 @@ export class EmailService {
 
       if (this.configService.get('email.smtp.host')) {
```

Comment:
If this is false, will it not send?

Reply:
Should it return null or something else?